### ISSUE TITLE
Issue with created page

### ISSUE THREAD

**Issue by TheRedEclipse** [rainlab/pages-plugin#335]
Hello, I just created a few static pages with same layout and faced with a problem - I can't open one of them. When I trying to open it says "The requested object was not found.". Can you tell me how to fix this page?

Thanks

**Comment by TheRedEclipse** [comment]
Not sure if its right what you're looking for, but when I attempt to open that page browser console says:

jquery.min.js?v431:4 POST http://domain.com/backend/rainlab/pages 500 (Internal Server Error)

**Comment by LukeTowers** [comment]
@TheRedEclipse check your error logs, or open the network tab in developer tools in chrome and click on the request that gets the 500 response and see what the full response from the server is.

**Comment by TheRedEclipse** [comment]
Oh, sorry. If I got you right this time, there numerous errors about missing default layout, but last is dated on 9 april.

`Cms\Classes\CmsException: The layout 'default' is not found in /home/admin/web/domain.com/public_html/modules/cms/classes/Controller.php:274
#0/home/admin/web/domain.com/public_html/modules/cms/classes/Controller.php(213): Cms\Classes\Controller→runPage(Object(Cms\Classes\Page))#1/home/admin/web/domain.com/public_html/modules/cms/classes/CmsController.php(50): Cms\Classes\Controller→run('js/map.js')#2[internal function]:Cms\Classes\CmsController→run('js/map.js')#3/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): call_user_func_array(Array, Array)#4/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(45): Illuminate\Routing\Controller→callAction('run', Array)#5/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(212): Illuminate\Routing\ControllerDispatcher→dispatch(Object(Illuminate\Routing\Route), Object(Cms\Classes\CmsController), 'run')#6/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(169): Illuminate\Routing\Route→runController()#7/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(658): Illuminate\Routing\Route→run()#8/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(30): Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))#9/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(41): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))#10/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Routing\Middleware\SubstituteBindings→handle(Object(Illuminate\Http\Request), Object(Closure))#11/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))#12/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))#13/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\View\Middleware\ShareErrorsFromSession→handle(Object(Illuminate\Http\Request), Object(Closure))#14/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))#15/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(63): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))#16/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Session\Middleware\StartSession→handle(Object(Illuminate\Http\Request), Object(Closure))#17/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))#18/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))#19/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse→handle(Object(Illuminate\Http\Request), Object(Closure))#20/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))#21/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))#22/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Cookie\Middleware\EncryptCookies→handle(Object(Illuminate\Http\Request), Object(Closure))#23/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))#24/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))#25/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(660): Illuminate\Pipeline\Pipeline→then(Object(Closure))#26/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(635): Illuminate\Routing\Router→runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))#27/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(601): Illuminate\Routing\Router→runRoute(Object(Illuminate\Http\Request), Object(Illuminate\Routing\Route))#28/home/admin/web/domain.com/public_html/vendor/october/rain/src/Router/CoreRouter.php(20): Illuminate\Routing\Router→dispatchToRoute(Object(Illuminate\Http\Request))#29/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(176): October\Rain\Router\CoreRouter→dispatch(Object(Illuminate\Http\Request))#30/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(30): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))#31/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(46): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))#32/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode→handle(Object(Illuminate\Http\Request), Object(Closure))#33/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))#34/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))#35/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(151): Illuminate\Pipeline\Pipeline→then(Object(Closure))#36/home/admin/web/domain.com/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(116): Illuminate\Foundation\Http\Kernel→sendRequestThroughRouter(Object(Illuminate\Http\Request))#37/home/admin/web/domain.com/public_html/index.php(43): Illuminate\Foundation\Http\Kernel→handle(Object(Illuminate\Http\Request))#38{main}`

**Comment by LukeTowers** [comment]
Sounds like you assigned them to a layout and then either removed or renamed the layout file

**Comment by TheRedEclipse** [comment]
Indeed, for test purposes I've renamed it and forgot to change it back, didn't knew it must have. But all of them was created without default.htm and work fine except one.

**Comment by TheRedEclipse** [comment]
Nope, still face to this issue. I tried to create few extra pages and these works properly.

I don't know what exactly cause this problem to start a pull request, so I rather start project from scratch again before much of changes.
